navigation/ca_move_base/maps/empty.yaml

```diff
@@ -2,5 +2,5 @@ free_thresh: 0.196
 image: empty.pgm
 negate: 0
 occupied_thresh: 0.65
-origin: [-6.8999999999999915, -5.8999999999999915, 0.0]
+origin: [0.0, 0.0, 0.0]
```

Comment:
Ramiro created a PR (#104) fixing this. It would be great if you can review it, and ping him to address the remaining task. :smile: